pipelines/controller-cli/tests/test_main.py

```diff
@@ -0,0 +1,89 @@
+import contextlib
+import io
+import sys
+import unittest
+from unittest import mock
+
+from src.main import main, status
+
+
+class TestControllerCLI(unittest.TestCase):
+
+    TEST_URL = "http://localhost:9004"
```

Comment:
Just to confirm, with the `request_mock` added now, we do not need a controller server running on port 9004 for these tests to pass, correct?

Reply:
Correct.

Reply:
Done